tests/test_upstream_issue.py

```diff
@@ -212,6 +212,40 @@ def test_github_issues_no_token(self,
         self.mock_github_repo.get_issue.assert_not_called()
         self.mock_github_issue.get_comments.assert_not_called()
 
+    @mock.patch('sync2jira.intermediary.Issue.from_github')
+    @mock.patch(PATH + 'Github')
+    @mock.patch(PATH + 'get_all_github_data')
+    def test_github_issues_no_token(self,
+                                    mock_get_all_github_data,
+                                    mock_github,
+                                    mock_issue_from_github):
+        """
+        This function tests 'github_issues' function with a filter including multiple labels
+        """
+        # Set up return values
+        self.mock_config['sync2jira']['filters']['github']['org/repo']['labels'].extend(['another_tag', 'and_another'])
+        mock_github.return_value =  self.mock_github_client
+        mock_get_all_github_data.return_value = [self.mock_github_issue_raw]
+        mock_issue_from_github.return_value = 'Successful Call!'
+
+        # Call the function
+        list(u.github_issues(
+            upstream='org/repo',
+            config=self.mock_config
+        ))
+
+        # Assert that calls were made correctly
+        try:
+            mock_get_all_github_data.assert_called_with(
+                'https://api.github.com/repos/org/repo/issues?labels=custom_tag%2Canother_tag%2Cand_another&filter1=filter1',
+                {'Authorization': 'token mock_token'}
+            )
+        except AssertionError:
+            mock_get_all_github_data.assert_called_with(
+                'https://api.github.com/repos/org/repo/issues?filter1=filter1&labels=custom_tag%2Canother_tag%2Cand_another',
+                {'Authorization': 'token mock_token'}
+            )
```

Comment:
I realize that you copied this construct from somewhere else, but I think that a different approach would be better here.  We don't actually care what the whole argument looks like, we care only that the argument contains the correct `labels` query parameter value.  So, the assertion should check just for that, in which case we don't need the `try` block or the second assertion.

So, something like this:
```suggestion
        # Assert that calls were made correctly
        expected = 'labels=custom_tag%2Canother_tag%2Cand_another'
        actual = mock_get_all_github_data.args[0]
        self.assertIn(expected, actual)
```

Reply:
Also, since we are now intentionally mutating the configuration, consider adding an assertion which confirms that after the call(s):
```python
        actual = self.mock_config['sync2jira']['filters']['github']['org/repo']['labels']
        expected = 'custom_tag,another_tag,and_another'
        self.assertEqual(actual, expected)
```

Reply:
Yep, had no idea how to go about this so just copied what was already there - so thank you very much for your suggestions, applied both.